StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -453,24 +453,24 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
     // MARK: - Networking
 
     func saveAccountsToLink(
-        emailAddress: String,
-        phoneNumber: String,
-        country: String,
+        emailAddress: String?,
+        phoneNumber: String?,
+        country: String?,
         selectedAccountIds: [String],
         consumerSessionClientSecret: String?,
         clientSecret: String
     ) -> Future<FinancialConnectionsSessionManifest> {
         var body: [String: Any] = [
             "client_secret": clientSecret,
-            "email_address":
-                emailAddress
-                .trimmingCharacters(in: .whitespacesAndNewlines)
-                .lowercased(),
-            "phone_number": phoneNumber,
-            "country": country,
-            "locale": Locale.current.identifier,
             "selected_accounts": selectedAccountIds,
+            "expand": ["active_auth_session"],
         ]
+        body["email_address"] = emailAddress?
+            .trimmingCharacters(in: .whitespacesAndNewlines)
+            .lowercased()
+        body["phone_number"] = phoneNumber
+        body["country"] = country
+        body["locale"] = (phoneNumber != nil) ? Locale.current.identifier : nil
```

Comment:
any specific reason for not sending locale for all requests? Existing js logic?

Reply:
yes, just using Stripe.js precedent, this is the logic:

https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/eccfa561/src/linkedAccounts/inner/components/panes/LinkLogin/useSaveToNetworkAndLink.ts#L172